bootstrap/http/src/main/kotlin/com/studentcenter/weave/bootstrap/meeting/dto/MeetingTeamLocationResponse.kt

```diff
@@ -0,0 +1,30 @@
+package com.studentcenter.weave.bootstrap.meeting.dto
+
+import com.studentcenter.weave.domain.meeting.enums.Location
+
+data class MeetingTeamLocationResponse(
+    val locations: List<MeetingTeamLocationDto>
+) {
+
+    data class MeetingTeamLocationDto(
+        val name: String,
+        val displayName: String,
+        val isCapitalArea: Boolean
```

Comment:
p4) tailing comma 체크해주세요!
```suggestion
        val isCapitalArea: Boolean,
```